lib/plugins/wallet/bitgo/bitgo.js

```diff
@@ -8,35 +9,43 @@ const pjson = require('../../../../package.json')
 const userAgent = 'Lamassu-Server/' + pjson.version
 
 const NAME = 'BitGo'
+const SUPPORTED_COINS = ['BTC', 'ZEC', 'LTC', 'BCH']
 
 function buildBitgo (account) {
   const env = account.environment === 'test' ? 'test' : 'prod'
   return new BitGo.BitGo({ accessToken: account.token, env, userAgent: userAgent })
 }
 
-function getWallet (account) {
+function getWallet (account, cryptoCode) {
   const bitgo = buildBitgo(account)
-  const coin = account.environment === 'test' ? 'tbtc' : 'btc'
+  const coin = account.environment === 'test' ? `t${cryptoCode.toLowerCase()}` : cryptoCode.toLowerCase()
 
-  return bitgo.coin(coin).wallets().get({ id: account.walletId })
+  return bitgo.coin(coin).wallets().get({ id: account[`${cryptoCode}WalletId`] })
```

Comment:
This feels awkward, thoughts on how to improve it?

Reply:
I think this is good for now.